config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-presubmits-main.yaml

```diff
@@ -118,19 +118,21 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
-      env:
-      - name: GITHUB_TOKEN
-        valueFrom:
-          secretKeyRef:
-            name: cluster-lifecycle-github-token
-            key: cluster-lifecycle-github-token
-    volumes:
-    - name: cluster-lifecycle-github-token
-      secret:
-        secretName: cluster-lifecycle-github-token
-        items:
-        - key: cluster-lifecycle-github-token
-          path: cluster-lifecycle-github-token
+        volumeMounts:
+        - name: cluster-lifecycle-github-token
```

Comment:
I think a `mountPath` is also required?
That being said, this volume doesn't appear to be necessary, the secret value is supplied via the env separately so we can just remove the unused volume and mounts.
The same applies to the other similar jobs.

Reply:
there was a unit test error coming up if the volume was available but never mounted... even though it was being consumed into the env ... I added the volumeMounts but left out the path as having secrets mounted in a container path didnt make sense

Reply:
The env does not consume or reference the volume, it references the secret directly so the unused volume can be removed.